lib/src/box.dart

```diff
@@ -166,5 +166,29 @@ class Box<T> {
     return _getMany(() => checkObxPtr(bindings.obx_box_get_all(_cBox), "failed to get all objects from box", true));
   }
 
+  // same as calling maxCount with limit := 0
+  int count() {
+    return maxCount(limit: 0);
+  }
+
+  int maxCount({int limit}) {
+    Pointer<Uint64> _count = Pointer<Uint64>.allocate();
+    checkObx(bindings.obx_box_count(_cBox, limit, _count));
+    return _count.load<int>();
+  }
+
+  bool isEmpty() {
+    Pointer<Uint8> _isEmpty = Pointer<Uint8>.allocate();
+    checkObx(bindings.obx_box_is_empty(_cBox, _isEmpty));
+    return _isEmpty.load<int>() > 0 ? true : false;
```

Comment:
`Pointer<T>` needs manual freeing. You can have a look at other code in `Box` to see how it's used.

In this example, it would look sth. like:
```dart
bool isEmpty() {
    Pointer<Uint8> _isEmpty = Pointer<Uint8>.allocate();
    try {
        checkObx(bindings.obx_box_is_empty(_cBox, _isEmpty));
        return _isEmpty.load<int>() > 0 ? true : false;
    } finally {
        _isEmpty.free();
    }
}
```